underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/page/PageRenderer.kt

```diff
@@ -75,7 +98,7 @@ class PageRenderer(private val linker: Linker,
                                     .attr("crossorigin", "anonymous"),
                             script()
                                     .with(UnescapedText("""
-                                        (function () {
+                                        (function renderMathElements() {
```

Comment:
Nice, when I added this function in a hurry, just copy-pasted the code from some MathJaxx resource. Thanks for taking care of this one as well!